samples/buildSrc-test-junit4/README.md

```diff
@@ -0,0 +1,7 @@
+buildSrc-test-junit4
+====================
+
+Sample project setup to show how you can test your _buildSrc_ custom build logic using
+Junit4 Unit tests.
+
+`./gradlew hello` this task uses a build helper function from _buildSrc_. The code inside _buildSrc_ will be compiled and tested prior to the execution of this task.
```

Comment:
Link the `buildSrc` in that markdown file: `[buildSrc](buildSrc/)`

Reply:
done